bin/nwbuild

```diff
@@ -54,9 +54,10 @@ if(!files) {
 }
 
 var options = {
+    appName: argv.name,
```

Comment:
Cool, this still falls back to the name in `package.json` if it doesn't exist yeah?

Reply:
of course. https://github.com/nwjs-community/nw-builder/blob/master/lib/index.js#L185